src/libopensc/opensc.h

```diff
@@ -248,6 +248,7 @@ typedef struct sc_security_env {
 	unsigned long flags;
 	int operation;
 	unsigned long algorithm, algorithm_flags;
+	size_t key_size;
```

Comment:
please make a comment that this is in bits rather than bytes

Reply:
Changed the field name to key_size_bits.